src/Toolbar/ToolbarItem.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { ElementType, FC, HTMLAttributes, MouseEvent } from 'react';
+import classNames from 'classnames';
+import { Icon } from '../Icon/Icon';
+
+type HTMLElementWithDisabled<T> = Partial<T> & { disabled?: boolean };
+
+export interface ToolbarItemProps
+  extends HTMLElementWithDisabled<HTMLAttributes<HTMLElement>> {
+  tag?: ElementType;
+  classname?: string;
+  active?: boolean;
+  url?: string;
+  label?: string;
+  srText?: string;
+  iconName?: string;
+  isIconSmall?: boolean;
+  alert?: boolean;
+  badge?: number;
+  onLinkClick?: (event: MouseEvent<HTMLAnchorElement>) => void;
+}
+
+export const ToolbarItem: FC<ToolbarItemProps> = ({
+  active = false,
+  badge,
+  url,
+  srText,
+  iconName = 'it-comment',
```

Comment:
I think this prop should be mandatory for a `ToolbarItem` - therefore the default value is not required.